Src/BootCamp.Chapter/Lesson10.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson10
+    {
+        public static string EncryptCypher(string message, int shift)
+        {
+            if (shift < 0)
+            {
+                shift = Math.Abs(shift + 95);
+            }
+
+            var messageBytes = Encoding.ASCII.GetBytes(message);
+
+            for (int i = 0; i < message.Length; i++)
+            {
+                var currentCharByte = messageBytes[i];
+                byte newByte = Convert.ToByte((((currentCharByte - 32) + shift) % 95) + 32);
```

Comment:
use constants instead hard coded values.

Reply:
Created constants CypherLength and Unused